src/WorkflowExecuter/Services/WorkflowExecuterService.cs

```diff
@@ -295,6 +396,15 @@ private async Task<bool> DispatchTask(WorkflowInstance workflowInstance, TaskExe
             return await _workflowInstanceRepository.UpdateTaskStatusAsync(workflowInstance.Id, taskExec.TaskId, TaskExecutionStatus.Dispatched);
         }
 
+        private async Task<bool> ExportRequest(WorkflowInstance workflowInstance, TaskExecution taskExec, string[] exportDestinations, IList<string> dicomImages, string correlationId)
+        {
+            var exportRequestEvent = EventMapper.ToExportRequestEvent(dicomImages, exportDestinations, taskExec.TaskId, workflowInstance.Id, correlationId);
+            var jsonMesssage = new JsonMessage<ExportRequestEvent>(exportRequestEvent, MessageBrokerConfiguration.WorkflowManagerApplicationId, exportRequestEvent.CorrelationId, Guid.NewGuid().ToString());
+
+            await _messageBrokerPublisherService.Publish(ExportRequestRoutingKey, jsonMesssage.ToMessage());
```

Comment:
I see that `ExportRequestRoutingKey` is assigned to `configuration.Value.Messaging.Topics.ExportRequestPrefix` so this may not have the correct routing key. Each export service should have its own, and (I guess) this would be coming from the workflow definition.

Reply:
Good shout, I have added this, I believe it will only use dicom web currently so I have set the routing key to be that for now.